README.md

```diff
@@ -1,59 +1,100 @@
-For questions write a mail to <berlin@berlin.freifunk.net>, open a discussion with 
+If you have any questions, send an email to <berlin@berlin.freifunk.net>, open a discussion with 
 @freifunk-berlin/firmware or come to our weekly meetings.
-If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
+If you find bugs, please report them at: https://github.com/freifunk-berlin/firmware/issues
 
 ## Development
 
 ### Info
 
 For the Berlin Freifunk firmware we use vanilla OpenWrt with additional patches and packages.
-The Makefile automates firmware creation (applies patches, integrates custom packages and uses
-the Imagebuilder of OpenWrt).
+The Makefile automates the build of the firmware (applies patches, integrates custom packages and uses the
+the ImageBuilder of OpenWrt).
 
 The idea is to download OpenWrt via git, patch it with all patches in the `patches/` folder
-and configure it according to the configuration snippets in `configs/`. Then use the OpenWrt
-buildsystem to build all packages and the Imagebuilder. The packages to assemble into the final
-image for the router are taken from the imageflavors defined in `packagelist/`.
+patch it and configure it according to the configuration snippets in `configs/`. Then use the OpenWrt
+build system to build all packages and the image builder. The packages that will be assembled to the final image for the router
+image for the router are taken from the image flavors defined in `packagelist/`.
 
-### Build Prerequisites
+### Build prerequisites
 
 Please take a look at the [OpenWrt documentation](https://openwrt.org/docs/guide-developer/build-system/install-buildsystem?s[]=prerequisites#prerequisites)
-for a complete and uptodate list of packages for your operating system. 
+for a complete and up-to-date list of packages for your operating system. 
 
-On Ubuntu/Debian:
+Alpine:
 ```
-apt-get install git build-essential libncurses5-dev zlib1g-dev gawk time \
-  unzip libxml-perl flex wget gawk gettext quilt python libssl-dev
+apk add asciidoc bash bc binutils bzip2 cdrkit coreutils diffutils \
+findutils flex g++ gawk gcc gettext git grep intltool libxslt \
+linux-headers make ncurses-dev openssl-dev patch perl \
+python2-dev python3-dev rsync tar unzip util-linux wget zlib-dev
 ```
 
-On openSUSE:
+Arch / Manjaro:
+```
+# Essential prerequisites
+pacman -S --needed base-devel bash bzip2 git libelf libxslt ncurses \
+openssl python2 time unzip util-linux wget zlib
+ 
+# Optional prerequisites, depend on the package selection
+pacman -S --needed asciidoc help2man intltool perl-extutils-makemaker
+```
+
+CentOS / Fedora:
+```
+sudo dnf --skip-broken install bash-completion bzip2 gcc gcc-c++ git \
+make ncurses-devel patch perl-Data-Dumper perl-Thread-Queue python2 \
+python3 rsync tar unzip wget perl-base perl-File-Compare \
+perl-File-Copy perl-FindBin diffutils which
+```
+
+Debian / Ubuntu:
+```
+sudo apt install build-essential ccache ecj fastjar file g++ gawk \
```

Comment:
* I'm quite sure that we don't need all the java-stuff .
* 'ccache' seems optional
* python*-dev ?

in contrast to the current list, you new seems very "bloated"

Reply:
I've used the packages listed by OpenWrt (https://openwrt.org/docs/guide-developer/build-system/install-buildsystem#examples_of_package_installations)